examples/interaction/tools/position_toolbar.py

```diff
@@ -1,3 +1,14 @@
+''' A basic scatter plot with the toolbar positioned below the plot,
+outside of the axes, titles, etc.
+This example shows a way to set the toolbar's location relative to the plot,
+outside of the axes, titles, etc.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.toolbar_location, bokeh.plotting.figure.toolbar_sticky, bokeh.plotting.figure.scatter, bokeh.plotting.show
+    :refs: :ref:`_ug_interaction_tools_toolbar`
```

Comment:
The leading underscore is Sphinx syntax for defining a reference, but actually using a reference, it needs to not have it:
```suggestion
    :refs: :ref:`ug_interaction_tools_toolbar`
```